app/services/candidate_interface/invite_application.rb

```diff
@@ -2,9 +2,10 @@ module CandidateInterface
   class InviteApplication
     attr_reader :application_form, :application_choice
 
-    def initialize(application_form:, application_choice:)
+    def initialize(application_choice:, application_form: nil, invite: nil)
```

Comment:
Possibly out of scope, but we shouldn't allow this to be initialized with an application form if the choice is a required param. The application form must always been the one associated with the application_choice.  
`@application_form = application_choice.application_form.`